ui/test/kapacitor/components/TasksTable.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {shallow} from 'enzyme'
+
+import TasksTable from 'src/kapacitor/components/TasksTable'
+import {TaskRow} from 'src/kapacitor/components/TasksTable'
+
+import {source, kapacitorRules} from 'test/resources'
+
+describe('Kapacitor.Components.TasksTable', () => {
+  describe('rendering', () => {
+    const props = {
```

Comment:
Using the `setup` function can eliminate a lot of this props duplication.

Reply:
agreed. i did it this way in anticipation of using multiple instances of `mount` for the end-to-end test, whereby i would've needed a separate `setup` for that mount. but then i also feel like i currently like knowing within each block whether i did shallow/mount/render. something to discuss as we converge on patterns, for sure